server/src/helpers/send.email.message.ts

```diff
@@ -37,12 +37,6 @@ export const createUserMessage = (username: string, token: string): string => {
                     </td>
                 </tr>
             </tbody>
-            <tfoot align="center" style="position: absolute; bottom: 0; width: 100%;
```

Comment:
why did u rm it?

Reply:
Because on email this footer doesn't fit there and my team solved to delete its